src/main/kotlin/org/wycliffeassociates/otter/jvm/app/Main.kt

```diff
@@ -35,18 +26,24 @@ fun main(args: Array<String>) {
 }
 
 private fun initApp() {
-    ImportLanguages(
-            File(ClassLoader.getSystemResource("langnames.json").toURI()),
-            Injector.languageRepo)
-            .import()
-            .onErrorComplete()
-            .subscribe()
+    Injector.preferences.setAppInitialized(false)
+    if (!Injector.preferences.getAppInitialized()) {
```

Comment:
I know you're technically getting a value, but appInitialized() might be a better name for the purpose of the not?

Reply:
Sure, that's sounds better to me as well